src/stripeWebhooksView.ts

```diff
@@ -0,0 +1,86 @@
+import * as vscode from 'vscode';
+import {StripeDaemon} from './daemon/stripeDaemon';
+import {StripeTreeItem} from './stripeTreeItem';
+import {StripeTreeViewDataProvider} from './stripeTreeViewDataProvider';
+// import {ThemeIcon} from 'vscode';
+import {WebhookEndpointsListRequest} from './rpc/webhook_endpoints_list_pb';
+
+export class StripeWebhooksViewProvider extends StripeTreeViewDataProvider {
+  private stripeDaemon: StripeDaemon;
+  private endpointItems: StripeTreeItem[];
+
+  constructor(stripeDaemon: StripeDaemon) {
+    super();
+    this.stripeDaemon = stripeDaemon;
+    this.endpointItems = [];
+  }
+
+  buildTree(): Promise<StripeTreeItem[]> {
+    const items = [];
+
+    // DX-7014
+    // const createEndpoint = new StripeTreeItem('Create a new webhook endpoint', {
+    //   commandString: 'createWebhookEndpoint',
+    //   iconPath: new ThemeIcon('add'),
+    //   tooltip: 'Create a new webhook endpoint',
+    //   contextValue: 'createWebhookEndpoint',
+    // });
+    // items.push(createEndpoint);
+
+    if (this.endpointItems.length > 0) {
+      const endpointsRootItem = new StripeTreeItem('All webhook endpoints');
+      endpointsRootItem.children = this.endpointItems;
+      endpointsRootItem.expand();
```

Comment:
Does this mean this will start out expanded? I'm worried that it'll be too overwhelming for cases where the endpoints have all of the events enabled. Could we start this off as collapsed?